templates/main.html

```diff
@@ -11,16 +11,16 @@
         <button class='btn play' rv-on-click="togglePlay" rv-disabled="context.isDisconnected < state.current" rv-title="playText < context.state.current">
           <span rv-class="playIconClass < context.state.current"></span>
         </button>
-        <button class='btn step-over' title="Step over">
+        <button class='btn step-over' rv-on-click="stepOver" rv-disabled="context.isDisconnected < state.current" title="Step over">
```

Comment:
I haven't tried this PR out locally (I will though). Should this be enabled only when **paused** perhaps?

I'm thinking you might trigger a bug if you click it when the program is running. 

As I said will try out locally.